### ISSUE TITLE
Accented (Hungarian) characters with encryption

### ISSUE THREAD

**Issue by janesz04** [dompdf/dompdf#2839]
Hello!

I use the latest dompdf, along with the DejaVu Sans font set, so the accents are correct and everything works. However, if I want to encrypt, more precisely so that it cannot be edited, just opened and printed, the whole pdf will fall apart.

If I try with a different font, e.g. Helvetica, you can find the accented letters fall out of it. I read here that this problem has already been solved in principle, but it still doesn't work for me, even though it would be simple.

Thank you in advance for your help!

`$dompdf->get_canvas()->get_cpdf()->setEncryption('','password',array('print'));`

**Comment by bsweeney** [comment]
Did you resolve your issue? A quick test and the PDF looks OK. If you're still having issue provide more details and we'll look into it.

**Comment by janesz04** [comment]
Yes, it worked, thank you. It already worked with the latest version 
1.2.1, only I was working with the previous version.

2022.03.28. 19:26 keltezéssel, Brian Sweeney írta:
